### DISCUSSION TITLE
Schematic hand/selection tools fusion

### DISCUSSION THREAD

**Discussion by Al532** [opentoonz/opentoonz#6707] [Ideas]
IMO, in the schematic view, hand tool and selection tool should be an unique tool. When clicking on an item, it would work as the selection tool, when clicking everywhere else it would work as the hand tool. That would be very handy to navigate with a mouse (the mouse wheel being used for the zoom).

**Comment by Al532** [comment]
Hi!
I'm coming back to my OT project, after almost one year off. I see this feature request had been added to the 1.5 and is checked, should it mean that it's include in 1.5 stable version? In fact, it isn't. I am missing something about release schedule?

**Comment by RattusaurusRex** [comment]
It is under "Unresolved" and is not checked. 
![image](https://user-images.githubusercontent.com/60277069/115343445-c9d45400-a160-11eb-9884-c639ec71c843.png)

**Comment by Al532** [comment]
Ah, thanks! Sorry I was looking at the #3597, which is the Release Schedule number.
